lib/Controller/Display.php

```diff
@@ -716,6 +716,8 @@ function grid(Request $request, Response $response)
                     $display->commercialLicenceDescription = __('Display is not licensed');
             }
 
+            $display->commercialLicenceDescription .= ' (' . __('The status will be updated with each Commercial Licence check') . ')';
```

Comment:
Please can we future proof this by only showing it if the player clientCode < 400 (i.e. version 4 players shouldn't show it).